db/migrate/20190708012000_create_institutions_archive.rb

```diff
@@ -0,0 +1,9 @@
+class CreateInstitutionsArchive < ActiveRecord::Migration
+  def up
+    execute "CREATE TABLE institutions_archives AS TABLE institutions WITH NO DATA;"
```

Comment:
What kind of indexes should this table have?

Reply:
Changed sql to create table so that it inherits `institutions`'s defaults, constraints, and indexes